src/daft-core/src/array/ops/list.rs

```diff
@@ -118,6 +118,61 @@ pub fn get_slices_helper(
     .into_series())
 }
 
+fn get_chunks_helper(
```

Comment:
Sounds good, added some docs. I also switched around the argument list while writing the docs because there was a more natural order to them: input/parent values (flat_child, field, validity) -> input/expression argument (size) -> "output"/result of preprocessing (elements to skip and new offsets)